db/fixtures/ae_datastore/Deployment/System/Request.class/__methods__/pre_deployment.rb

```diff
@@ -0,0 +1,82 @@
+require 'net/ssh'
+
+LOCAL_BOOK = 'local_book.yaml'
+COMMANDS = ['yum list installed git || yum install git',
+            'yum list installed epel-release || yum install epel-release',
+            'test -d /tmp/openshift-ansible || ' \
+            'git clone https://github.com/openshift/openshift-ansible.git /tmp/openshift-ansible',
+            'yum list installed ansible || yum install ansible',
+            'yum list installed pyOpenSSL ||yum install pyOpenSSL',
+            "mv /home/#{$evm.root['user']}/inventory.yaml /tmp/openshift-ansible"
+           ]
```

Comment:
@dkorn remember that this may change for atomic (image to pull/run instead of rpms)